main.tf

```diff
@@ -44,9 +45,11 @@ resource "ibm_database" "postgresql_db" {
     cpu {
       allocation_count = var.member_cpu_count
     }
-
-    members {
-      allocation_count = var.members
+    dynamic "members" {
+      for_each = var.member_enabled ? [] : [1]
```

Comment:
See comment below - I think we should be able to set the number of member to 1 if the variable remote_leader_crn is set - as opposed to adding an extra variable in the module's interface

Reply:
Made the required changes.